source/_components/cover.rflink.markdown

```diff
@@ -126,6 +126,41 @@ devices:
           type: [list, string]
 {% endconfiguration %}
 
+### Setting up a KAKU ASUN-650 device
+
+In RFLink, the ON and DOWN command are used to close the cover and the OFF and UP command are used to open the cover. The KAKU (COCO) ASUN-650 works the other way around, it uses the ON command to open the cover and the OFF command to close the cover. The RFLink cover component has an extra property named `type` that takes 2 values:
+- standard: Do not invert the on/off commands sent to the RFLink device.
+- inverted: Invert the on/off commands sent to the RFLink device.
+
+The following configuration example shows how to use the `type` property:
+
+```yaml
+# Example configuration.yaml entry that shows how to
+# use the type property.
+cover rflink:
+  - platform: rflink
+    device_defaults:
```

Comment:
You can leave this away by default it's `false`

Reply:
Ok, I removed the complete 'device_defaults' section. I'm not that skilled with Home Assistant and learning everyday ;-)